src/cms/contentful.ts

```diff
@@ -1,12 +1,21 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 import {
-  CreateClientParams as ContentfulClientParams,
+  type CreateClientParams as ContentfulClientParams,
   createClient,
-  ContentfulClientApi,
+  type ContentfulClientApi,
+  type Entry,
+  type EntrySkeletonType,
+  EntryQueries,
 } from 'contentful';
 
+import { HarmonizedOutput } from '../@types';
 import { AbstractAgnosticCMSHarmonizerClient } from '../index.abstract';
 
+enum ContentfulResourceType {
+  ENTRY = 'Entry',
+  ASSET = 'Asset',
+}
```

Comment:
Use `const` `as const` please! `enum`s are known to have bad performance and confusing output while providing no value over `const`.

```suggestion
const ContentfulResourceType  = {
  ENTRY: 'Entry',
  ASSET: 'Asset',
} as const
```

Also, I think it is a good idea to decouple consts, magics and any other hard coded data from the implementation and put to a centralised place instead.

Reply:
Done!

About moving to a centralized place, as this is very specific to Contentful and not used out of that class (not needed to be exported to the user), I don't know if it needs to be moved. What place do you suggest @matyasjay ?

Reply:
Nah, I wasn't really aware of the scope. Fine for me to keep it here, thanks!